lmdeploy/turbomind/turbomind.py

```diff
@@ -87,6 +88,37 @@ def _update_tm_config(dst: TurbomindModelConfig, src: TurbomindEngineConfig):
     return TurbomindModelConfig.from_dict(dst_dict)
 
 
+def _compare_individual_gpu_memory():
+    logging.basicConfig(level=logging.INFO)
+
+    devices = os.getenv('CUDA_VISIBLE_DEVICES', '')
+    device_ids = list(map(int, devices.split(','))) if devices else []
+
+    try:
+        total_mem = []
+        free_mem = []
+
+        for i in range(torch.cuda.device_count()):
+            if device_ids and i not in device_ids:
+                continue
```

Comment:
```suggestion
        for i in range(tp):
```

Reply:
and `tp` should be passed as a parameter.

Reply:
> and `tp` should be passed as a parameter.

Ok you are right. I've fixed it. Thanks.